### PR TITLE
HHVM-136: Make hippo and phongo README/CONTRIBUTING files consistent

### PR DESCRIPTION
**Pull request by derickr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +38,7 @@ $ echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed
 ```
 
 We recommend using this extension in conjunction with our
-[userland library](https://github.com/mongodb/mongo-php-library),
+[userland library](https://github.com/mongodb-labs/mongo-php-library-prototype),
```
Comment (bjori) [inline README.md]: The previous version is correct

File: README.md [hunk 2]
```diff
@@ -3,18 +3,27 @@
...
+
+The driver is written to be a bare bone layer to talk to MongoDB, and
+therefore misses many convenience features. Instead, these convenience methods
+have been split out into a layer written in PHP, the
+[MongoDB Library](http://mongodb.github.io/mongo-php-library/).
+Using this library should be your preferred way of interacting with MongoDB.
+
+Please note that the new HHVM and PHP drivers implement a **different API**
+from the legacy driver at http://pecl.php.net/package/mongo, and hence, you
+will find it hard for other libraries like
```
Comment (bjori) [inline README.md]: "you will find it hard for other libraries" what does that mean?

File: README.md (hunk 1 above)
Comment (derickr) [inline README.md]: Indeed - it was a merge conflict. I'll fix it back :-)

File: README.md [hunk 3]
```diff
@@ -3,18 +3,27 @@
 [![Build Status](https://api.travis-ci.org/mongodb/mongo-php-driver.png?branch=master)](https://travis-ci.org/mongodb/mongo-php-driver)
 [![Coverage Status](https://coveralls.io/repos/mongodb/mongo-php-driver/badge.svg?branch=master&service=github)](https://coveralls.io/github/mongodb/mongo-php-driver?branch=master)
 
-The purpose of this driver is to provide a thin glue between MongoDB
-and PHP, implementing only fundamental and performance-critical components
+This is the low-level PHP driver for MongoDB. It implements the same interface
+as the HHVM driver for MongoDB. The documentation for both of them is the same,
```
Comment (bjori) [inline README.md]: This sounds like a chicken and egg problem. The PHP driver isn't implementing the HHVM interface as this suggests

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-# Contributing to pecl/MongoDB
+# Contributing to the PHP Driver for MongoDB
 
 
-## Building from VCS
+## Building from VCS (GitHub)
```
Comment (bjori) [inline CONTRIBUTING.md]: GitHub isn't a VCS.....

File: README.md (hunk 2 above)
Comment (derickr) [inline README.md]: Will fix that too.

**Pull request merged**